### PR TITLE
Add week 30 updates blog post

### PR DESCRIPTION
**Pull request by Vinai**

### CODE REVIEW

**Review by johnhughes1984 — APPROVED** [review]

File: src/pages/blog/mage-os-update-2022-week-30.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+layout: '@/layouts/post.astro'
+date: 2022-07-29
+title: Week 26-30 of 2022
```
Comment (sprankhub) [inline src/pages/blog/mage-os-update-2022-week-30.md:4]:
```suggestion
title: New Website, Mirror Distribution, Nightly Builds, Content & Community Updates
```

@rhoerr suggested to use more meaningful titles, which makes sense IMHO. I kind of dislike just listing the topics, but this is still better than a generic title.

File: src/pages/blog/mage-os-update-2022-week-30.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+### Mage-OS Magento® Mirror Distribution
+
+Quite some work went into stabilizing older releases of the mirror distribution. All supported releases install cleanly and quickly now.
+
+Also, we added release 2.3.7-p3 to the list of supported releases because it is still being used in some CI pipelines to test modules against it.
+Now all relevant versions can be installed from the <https://mirror.mage-os.org> composer repository.
+
+Thank you to Daniel Sloof, Flyingmana, and Damian Retzinger for helping to debug some of the issues! Your input was invaluable.
+
+Please join our [discord](http://chat.mage-os.org), say hi in the #tech channel, and join the conversation!
```
Comment (sprankhub) [inline src/pages/blog/mage-os-update-2022-week-30.md:30]:
```suggestion
Please join our [Discord](http://chat.mage-os.org), say hi in the #tech channel, and join the conversation!
```

File: src/pages/blog/mage-os-update-2022-week-30.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+Thank you to Anton, Willem, Simon, Daniel, and Flyingmana (and others whom I might have forgotten) for providing input on this topic.
+
+Once this change is implemented we will post details on how to install a build nightly release, and also how to build nightly releases yourself.
+
+
+## Organizational topics
+
+### Magento Open Source Content
+
+The content team continues to work on creating content for a Magento Open Source focused website. More content creators are very welcome! Every Monday at 4pm CET we meet to coordinate. Please join our [discord](http://chat.mage-os.org) and visit the #content channel to participate.
```
Comment (sprankhub) [inline src/pages/blog/mage-os-update-2022-week-30.md:51]:
```suggestion
The content team continues to work on creating content for a Magento Open Source focused website. More content creators are very welcome! Every Monday at 4pm CET we meet to coordinate. Please join our [Discord](http://chat.mage-os.org) and visit the #content channel to participate.
```

File: src/pages/blog/mage-os-update-2022-week-30.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+### Community
+
+The [Mage-OS Discord](http://chat.mage-os.org) is becoming more lively, which is lovely. Please join and introduce yourself, it is great to see people are interested in an independent Magento Open Source.
+
+Also, we want to have online meetups or town hall hangouts, soon. These will be a great place to get to know each other, ask questions, and hear about what is happening.
+
+Finally, we are using Discords audio and video channels to stream longer dev sessions about once a week. This usually is rather spontaneous, usually during the weekend. Keep an eye out for Vinai or others hanging out in the Watercooler chats, if you are interested to discuss technicalities, pair program, or just watch.
```
Comment (sprankhub) [inline src/pages/blog/mage-os-update-2022-week-30.md:59]:
```suggestion
Finally, we are using Discord's audio and video channels to stream longer dev sessions about once a week. This usually is rather spontaneous, usually during the weekend. Keep an eye out for Vinai or others hanging out in the Watercooler chats, if you are interested to discuss technicalities, pair program, or just watch.
```

Not a native speaker, but I think there needs to be an [apostrophe](https://www.dict.cc/?s=apostrophe) :D

File: src/pages/blog/mage-os-update-2022-week-30.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+
+That's all for today!  
+Enjoy the week, more news coming soon,
```
Comment (sprankhub) [inline src/pages/blog/mage-os-update-2022-week-30.md:63]:
```suggestion
Enjoy the week and watch out - more news coming soon!
```

**Review by rhoerr — APPROVED** [review]

**Pull request merged**